test/utils.ts

```diff
@@ -36,6 +36,11 @@ export const mockMwsFail = new MWS(
   new HttpClient(httpConfig, () => Promise.resolve({ data: '', headers: {} })),
 )
 
-export const parsingError = 'Expected an object, but received a string with value ""'
+export const mockParsingError = new MWS(
+  new HttpClient(httpConfig, () => Promise.resolve({ data: '<xml></xml>', headers: {} })),
+)
+
+export const parsingError = 'Start tag expected.'
+// export const parsingError = 'Expected an object, but received a string with value ""'
```

Comment:
Do we still need this comment here?

Reply:
Actually working on this branch now, I'll be removing this comment. Just wanted it there just as a comparison to what it previously was